src/test/ui/cmse-nonsecure-entry/params-on-stack.stderr

```diff
@@ -0,0 +1,5 @@
+error: <unknown>:0:0: in function entry_function i32 (i32, i32, i32, i32, i32): secure entry function requires arguments on stack
```

Comment:
Yeah this seems fine for an initial version of this feature, but the error should be moved into rustc eventually (we know some function ABI details already, and this seems fairly easy to check). We can add a note about that to the tracking issue.

Reply:
Added to the tracking issue.